framework/material/Navigator.kt

```diff
@@ -102,9 +102,17 @@ import mu.KotlinLogging
 
 object Navigator {
 
-    enum class Mode(val clicksToOpenItem: Int) {
-        BROWSER(2),
-        LIST(1),
+    sealed interface Behaviour {
+        val clicksToOpenItem: Int
+        val itemsAreFocusable: Boolean
+
+        class Browser(override val clicksToOpenItem: Int = 2): Behaviour {
+            override val itemsAreFocusable = true
+        }
+
+        class Menu(override val clicksToOpenItem: Int = 1): Behaviour {
+            override val itemsAreFocusable = false
+        }
```

Comment:
As discussed, we have Navigator `Behaviour` classes that allow users to modify whether items will be focused on click and how many clicks are required to open items.

Reply:
Looks good! It's on the thin side for now, but I think it's very extensible.